generic/tests/pktgen_perf.py

```diff
@@ -93,6 +93,31 @@ def _pin_vm_threads(node):
     for record in record_list.split():
         record_line += "%s|" % format_result(record)
 
+    # check module pktgen, install kernel-modules-internal package
+
+    check_cmd = "uname -r |grep el8"
+    if process.run(check_cmd, shell=True) or session.cmd(check_cmd):
```

Comment:
I am not sure if I understand this correctly: to install kernel-modules-internal package on both host and guest when one of host and guest are RHEL8?
But from the bz, it seems that the issue only occur on the system that the kernel newer than some version? So, you should only install the package for those system?
So it might like:
```
from functools import partial as Partial

def install_package(ver, session=None):
    output_cmd = Partial(process.system_output, timeout=timeout, shell=True)
    kernel_ver = "kernel-modules-internal-%s" % ver
    cmd_download = "cd /tmp && brew download-build %s --rpm" % kernel_ver
    cmd_install = "cd /tmp && rpm -ivh  %s.rpm --force --nodeps" % kernel_ver
    output_cmd(cmd_download).decode()
    cmd_clean = "rm -rf /tmp/%s.rpm" % kernel_ver
    if session:
        output_cmd = session.cmd_output
        local_path = "/tmp/%s.rpm" % kernel_ver
        remote_path = "/tmp/"
        vm.copy_files_to(local_path, remote_path)
    output_cmd(cmd_install)
    output_cmd(cmd_clean)

require_kernel_version = xxx
if (host_ver>require_kernel_version):
    install_package(host_ver)
if (guest_ver>require_kernel_version):
    install_package(guest_ver, session=session)
```

Reply:
"to install kernel-modules-internal package on both host and guest when one of host and guest are RHEL8?" yes , you understand correctly. and if run rhel7 VM on rhel8 host, we do not need install kernel-modeuls-internal package on vm . 

I remembered from rhel8 beginning, the pktgen modules was removed from kernel, I don't know the exactly version, I check latest all rhel8 kernel from different product, they are all existing kernel-modules-internal package.

Reply:
> "to install kernel-modules-internal package on both host and guest when one of host and guest are RHEL8?" yes , you understand correctly. and if run rhel7 VM on rhel8 host, we do not need install kernel-modeuls-internal package on vm .
> 
I think your code does not match what you describe here.

> I remembered from rhel8 beginning, the pktgen modules was removed from kernel, I don't know the exactly version, I check latest all rhel8 kernel from different product, they are all existing kernel-modules-internal package.

Reply:
I have update and tests are passed.

 (1/1) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.pktgen_perf.q35: PASS (152.01 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2020-07-15T05.32-e7f3027/results.html